_posts/2021-08-04-cockpit-250.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Cockpit 250
+author: martin
+date: '2021-08-04'
+tags: cockpit, machines
+slug: cockpit-250
+category: release
+summary: ''
```

Comment:
Can this either be removed or replaced by something different than empty string?

Reply:
Done.